docs/04-tables.md

```diff
@@ -1,17 +1,16 @@
 Tables
 ======
 
-A Sleeper instance contains one or more tables. Each a table has four important
+A Sleeper instance contains one or more tables. Each table has four important
 properties: a name, a schema, its own S3 bucket for storing data for that
 table, and its own state store for storing metadata about the table
-(this is normally stored in DynamoDB tables, each Sleeper table will have
-its own DynammoDB tables for storing the metadata). All other resources
+(this is normally stored in DynamoDB tables). All other resources
```

Comment:
Delete "(this is normally stored in DynamoDB tables)" as the S3 state store is now the default.

And it's no longer true that each table has "its own S3 bucket for storing data for that table".

Reply:
Updated, thanks